topics/assembly/metadata.yaml

```diff
@@ -65,10 +65,33 @@ material:
       - "You should use SPAdes or another more modern assembler than Velvet for actual assemblies now."
     contributors:
       - slugger70
+  -
+    title: "Unicycler Assembly"
+    type: "tutorial"
+    name: "unicycler-assembly"
+    zenodo_link: "https://doi.org/10.5281/zenodo.842795"
+    galaxy_tour: "yes"
+    hands_on: "yes"
+    slides: "yes"
+    questions:
+      - "How to perform an hybrid bacterial assembly with Unicycler"
+    objectives:
+      - "Evaluate Illumina read quality"
+      - "Perform a Small genome Assembly with Unicycler"
+      - "Evaluate the Quality of the Assembly with Quast"
+      - "Annotate the assembly with Prokka"
+    time_estimation: "4h"
+    key_points:
```

Comment:
Can you add some key-points or take-home messages? They help trainees (and also instructors :wink: ) at the end of the tutorial to sum up what learned during the tutorial